kinova_moveit/robot_configs/j2s6s300_moveit_config/config/j2s6s300.srdf

```diff
@@ -43,7 +43,7 @@
     <!--END EFFECTOR: Purpose: Represent information about an end effector.-->
     <end_effector name="end_effector" parent_link="j2s6s300_end_effector" group="gripper" />
     <!--VIRTUAL JOINT: Purpose: this element defines a virtual joint between a robot link and an external frame of reference (considered fixed with respect to the robot)-->
-    <virtual_joint name="world_joint" type="fixed" parent_frame="world" child_link="root" />
+    <virtual_joint name="world_to_base_virtual_joint" type="fixed" parent_frame="world_frame" child_link="root" />
```

Comment:
Should we do the same with this file ?
kinova_moveit/robot_configs/j2s6s300_moveit_config/config/j2s6s300.srdf

Reply:
done